kratos/python_scripts/python_solver.py

```diff
@@ -179,21 +179,22 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
                 tmp = KratosMultiphysics.Parameters("{}")
                 KratosMultiphysics.ReorderAndOptimizeModelPartProcess(model_part, tmp).Execute()
 
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading model part from mdpa file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Finished reading model part from mdpa file.")
 
         elif (input_type == "rest"):
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Loading model part from restart file.")
             RestartUtility(model_part, self._GetRestartSettings(model_part_import_settings)).LoadRestart()
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished loading model part from restart file.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Finished loading model part from restart file.")
 
         elif(input_type == "use_input_model_part"):
-            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Using already imported model part - no reading necessary.")
+            KratosMultiphysics.Logger.PrintInfo("self.__class__.__name__", "Using already imported model part - no reading necessary.")
 
         else:
             raise Exception("Other model part input options are not yet implemented.")
 
-        KratosMultiphysics.Logger.PrintInfo("ModelPart", model_part)
-        KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]:: ", "Finished reading model part.")
+        if self.echo_level > 0:
+            KratosMultiphysics.Logger.PrintInfo("ModelPart", model_part)
```

Comment:
I would always print this

Reply:
I thought again about this and probably it is a good idea, esp for large models this can be a lot and is probably not necessary

I assigned the @KratosMultiphysics/implementation-committee to decide over this

In the meantime can you please revert the other prints such that this PR only contains this change? Thx